OmgvaPOS/src/OrderManagement/Controller/OrderUpdateController.cs

```diff
@@ -0,0 +1,105 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using OmgvaPOS.DiscountManagement.Controller;
+using OmgvaPOS.HelperUtils;
+using OmgvaPOS.OrderManagement.DTOs;
+using OmgvaPOS.OrderManagement.Service;
+
+namespace OmgvaPOS.OrderManagement.Controller;
+
+[ApiController]
+[Route("order")]
+public class OrderUpdateController(IOrderService orderService, ILogger<DiscountController> logger) : ControllerBase
+{
+    private readonly IOrderService _orderService = orderService;
+    private readonly ILogger<DiscountController> _logger = logger;
```

Comment:
Please rename `OrderUpdateController` -> `OrderItemController`, OrderController should have all the CRUD operations for order, this one is used for OrderItems as I understand